### ISSUE TITLE
Can't put refined fuel into canisters

### ISSUE THREAD

**Issue by KlayTitan** [NFINIT-Development/AstroBlock#75]
As the title suggests you can't put refined fuel from thermal foundation into the galacticraft cansiters, which prevent me from making boosters for the rockets.

- The fluid transposer will not put Refined Fuel into the canisters
- there is no way to convert Refined Fuel into Galacticraft Fuel
- Cant make a booster from a refined fuel bucket
- Refined Fuel can be used in a fuel loader, which then converts it to Galacticraft fuel, but I can't get the fuel back out, at least with the methods I have tried so far.

The only real solution I see is to make the Galacticraft refinery, which as far as I can tell there is no quest for. I'm cool with this being and intended feature, but it seems a little silly to have a surplus of refined fuel, then suddenly have to make a new kind of fuel JUST to make the boosters to get to mars and beyond, when the refined fuel works in the rockets. 

Suggestions include Adding fluid dictionary entities for Fuel or just skipping refined fuel, and making Gcraft fuel the ONLY fuel. or adding recipes for refined fuel buckets for the boosters.

**Comment by Andr0oS** [comment]
You can put oil (found on the overworld) into a fuel refinery and get the galacticraft fuel. Agree on the need for a quest to point in the right direction, but there is a solution to this problem.

**Comment by Delerium** [comment]
Also for those who come here later on with this problem, when refined fuel gets put into a rocket, it becomes galacticraft fuel and can be stored in fuel canisters, so if you fill your rocket to 100% for the trip to earth and the moon, you should get enough reserve fuel to craft the boosters you need to get to mars.  I still agree that refined fuel should be convertible to galacticraft fuel with ore dictionary, or just used interchangeably though.

**Comment by herougan** [comment]
Yup - I tried many things but could not get any sort of conversion of refined fuel into fuel easily
Only by launching the rocket and collecting the remains with the canister will you get filled fuel canisters. Alternatively, you can do the refinery route - but I doubt that is the way the pack creator intended since the guide uses liquified coal. I think the best route now is either for galacticraft to better fix his machines to thermal and mekanism piping or simply that refined fuel becomes fuel easier

**Comment by knoxhack** [comment]
Umm

**Comment by mtbrenner** [comment]
Alien Villagers on the Moon also trade partially filled fuel canisters.
